src/main/java/dev/espi/ebackup/UploadUtill.java

```diff
@@ -0,0 +1,86 @@
+package dev.espi.ebackup;
+
+import com.jcraft.jsch.*;
+import org.apache.commons.net.ftp.FTP;
+import org.apache.commons.net.ftp.FTPClient;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+
+public class UploadUtill {
```

Comment:
I don't really understand why there's a new class here, could the methods please be moved back to UploadUtil